packages/client-core/src/admin/adminRoutes.tsx

```diff
@@ -64,12 +64,15 @@ const AdminTopBar = () => {
   }
 
   return (
-    <div className="flex h-16 w-full items-center justify-between  px-8 py-4">
-      <img
-        src="static/app_logo.svg"
-        alt="Napster Engine Logo"
-        className={`h-7 w-7${theme.value === 'light' ? ' invert' : ''}`}
-      />
+    <div className="pointer-events-auto flex h-16 w-full items-center justify-between px-8 py-4">
+      <a href="/dashboard">
```

Comment:
the dashboard doesn't exist in the core engine. it should redirect to `/` and that will redirect to `/dashboard`